### PR TITLE
Feat/implement new UI for nft list

### PR DESCRIPTION
**Pull request by NhutVoWakumo**
- Implement NFT collection table
- Implement NFT collection detail page that includes
    - Collection MetaData
    - All NFTs of this collection
    - Owned NFTs of this collection

### CODE REVIEW

File: src/app/components/TokenCircle.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const TokenCircle = ({
         return (
           <Avatar
             key={index}
-            className="absolute w-16 h-16 rounded-full"
+            className="absolute h-16 w-16 rounded-full"
```
Comment (tuhoangwkm) [inline src/app/components/TokenCircle.tsx]: we can use `size-16` instead of `h-16 w-16`

Comment (tuhoangwkm) [reply src/app/components/TokenCircle.tsx]: Please update the change in other places as well.

File: src/app/constants/erc1155ABI.ts [hunk 2]
```diff
@@ -0,0 +1,314 @@
+export const ERC1155ABI = [
```
Comment (tuhoangwkm) [inline src/app/constants/erc1155ABI.ts]: Move this to the abis folder too

File: src/app/components/TokenCircle.tsx (hunk 1 above)
Comment (NhutVoWakumo) [reply src/app/components/TokenCircle.tsx]: I found and fixed them all 😎

**Pull request merged**